lib/core/src/Ouroboros/Network/Client/Wallet.hs

```diff
@@ -348,6 +361,7 @@ chainSyncWithBlocks fromTip queue responseBuffer =
         -> P.ClientStNext n block (Tip block) m Void
     collectResponses respond blocks Zero = P.ClientStNext
         { P.recvMsgRollForward = \block tip -> do
+            traceWith tr $ MsgChainRollForward block
```

Comment:
Logging every `rollForward` sounds like something that would cause great slowness. Have you measured the syncing time against mainnet and a non-null tracer on the Byron wallets?

Reply:
These logs should't be visible per default since they are marked `Debug` — but perhaps still worth a check. Haven't tried yet…

Reply:
We had the case in the past, where simply having a logger would cause tremendous performance regression because the system still needs, on each block, to assess whether or not it should be logged.